MarkdownToLatex/MarkdownToLatex/FuncCalculator.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using MathNet.Symbolics;
+using System.Globalization;
+using Expr = MathNet.Symbolics.SymbolicExpression;
+
+namespace MarkdownToLatex {
+    /// <summary>This class is used for processing single variable functions.</summary>
+    public class FuncCalculator : Calculator<double, string> {
+
+        public double Calculate(){
+            try{
+                var func = Expr.Parse(this.Element);
+                return func.Compile(this.Var)(this.Param ?? 0);
```

Comment:
Wir müssen das hier schon zu LaTeX konvertieren, da das von `MdToTex` gerufen wird. In dem Sinne:  
``` C#
return $"f({this.Var})=" + Math.Round(func.Compile(this.Var)(this.Param ?? 0), 2).ToString(CultureInfo.InvariantCulture);
```